eox_tenant/signals.py

```diff
@@ -63,6 +63,22 @@ def _update_settings(domain):
         setattr(base_settings, key, value)
 
 
+def _repopulate_apps(apps):
+    """
+    After the initial loading of the settings, some djangoapps can override the AppConfig.ready() method
+    to alter the settings in a particular way.
+    In this case we need to run the app config again.
+    We delegate the decision to the specific tenant on the EDNX_TENANT_INSTALLED_APPS key.
+    If present, the key is passed in the apps argument
+    """
+    from django.apps.config import AppConfig
```

Comment:
importing this on the top of the file could cause any issue?

Reply:
I don't think it will cause an issue. I was only trying to make it load only when the tenant requires it.
But that logic is flawed since this way it will have to load again and again instead of just one time when the server starts.
I'll fix this now

Reply:
Done